### ISSUE TITLE
All collectors disappeared from WebUI after update to 2.1

### ISSUE THREAD

**Issue by JulioQc** [Graylog2/graylog2-server#2828]
## Expected Behavior
Collectors are still displayed in webUI as before

## Current Behavior
Collectors are all gone from WebUI but logs are still sent.

## Possible Solution
No idea

## Steps to Reproduce (for bugs)

1. Install 2.0.3 OVA
2. Configure some Collector+NXlog
3. Update to 2.1 OVA
4. Observe how the collectors all went missing in WebUI

## Context
Impossible to push new config to collectors now that they are not visible. 
Reverting to old REST API as per doc makes WebUI as a whole unreachable.

## Your Environment

* Graylog Version: 2.1 OVA
* Elasticsearch Version:
* MongoDB Version:
* Operating System:
* Browser version:

**Comment by mariussturm** [comment]
Do you see any errors in the sidecar logs? Did you also update to the latest sidecar?

**Comment by JulioQc** [comment]
Sidecar logs and nxlogs are clean.
Didn't see the collector update so I updated it... so now it's showing up but with a "Failing" in WebUI and no log in his path...

On the other hand, that collector update is such a mess imo. It doesn't actually update the current installed collector (why change the install path??) so I have to redo everything from scratch or almost. Although I greatly appreciate the tools offered, I am not impressed by this update. Should of stayed on my working 2.0 install.

**Comment by JulioQc** [comment]
Ok finally got it working (typo in yml). Now i'll have to spend the next 2 days updating the whole infra because of all these changes and the pain involved in with this collector update. But at least it works now *sigh*

**Comment by mariussturm** [comment]
http://docs.graylog.org/en/2.1/pages/upgrade/graylog-2.1.html#collector-sidecar

**Comment by JulioQc** [comment]
Yeah I tried before updating that and it caused more problem then anything. And I agree anyways collectors should be updated. 
I just don't get why it isn't managed directly within Graylog and the update isn't overwriting the previous install. Could be some limitation but I know of many software that do exactly that. 
Sorry for my tone earlier, I was just so upset when I realized the workload awaiting me. :(
